test/deploy/k8s/cortx-deployment.yml

```diff
@@ -53,5 +61,5 @@ spec:
           mountPath: /etc/cortx/solution
         - name: cortx-config
           mountPath: /etc/cortx
-        - name: cortx-secret
-          mountPath: /etc/cortx/secret
+        - name: podinfo
+          mountPath: /etc/podinfo
```

Comment:
We should avoid using pod in CORTX source hence this should be nodeinfo or just node. 
        - name: node
          mountPath: /etc/cortx/solution/node-config

Reply:
This would be addressed in next PR